### PR TITLE
Catch all PR to improve before tutorial

### LINKED ISSUE #13617: MOOSE Tutorial Updates
## Reason

Over the years the MOOSE tutorial has gained several part a, part b stuff. It is would be better for the flow to create new steps instead of sub-dividing.

There also needs to be a descriptive version on the main website; the slides are not intended to be followed outside of our live workshops.

Also, there are some more general improvements with respect to the language used.

### New (old) steps for tutorial

- [x] 1 (1): Simple Diffusion
- [x] 2 (2): Kernels (DarcyPressure -> Kernel)
- [x] 3: DarcyPressure -> ADKernel
- [x] 4 DarcyPressure -> ADDiffusion, discuss optimization and object inhertence (KernalGrad, KernalValue); 
- [x] 5 (3): Material
- [x] 6: Function (new; update PackedColumn to use functions instead of constants)
- [x] 7: Convert to ADMaterial; discuss getAD...; 
- [x] 8 (4): Velocity Aux (update to use VectorAuxVariable)
- [x] 9: (5): Heat Conduction (remove BC for later)
- [x] 10: ICs: Create and Initial condition that varies with vertical (radial) direction
- [x] 11 BCs; create HeatConductionOutflow 
- [x] 12: (6) Equation Coupling
- [x] 13: Add data interpolation functions to ADMaterial (material coupling)
- [x] 14: (7) Adaptivity (remove d)
- [x] 15: MeshModifiers/Generators (use current 7d)
- [x] 16 (8):  Postprocessors and VPPs (and use PP to compare out flow conditions)
- [ ] 17: Controls: use TimePeriod to disable stuff during BC ramp-up
- [x] 18 (9): Mechanics
- [x] 19 (10): MultiApps/Transfers
- [x] 20 (11): Actions

##  FEM, Shape Functions, Solver...
We need to break up these sections, dumping them all on the user at the same time is too much. It should be possible to sprinkle them in through the sections as we go, but we need to talk about how and where they fit in.

### Other Improvements

- [x] The solver pages should only cover NEWTON and PJFNK, adding JFNK and FD slides just
adds confusion, we can add numerical solve debugging section and then talk about FD.
- [x] There needs to be specific pages on preconditioning, I think it should be separate from the solver pages (i.e., doc/content/numerical/preconditioning.md)
- [ ] The movie for multi-scale displacement 500x should be 50x
- [x] Investigate the 'por_var' in current step10_micro.i
- [x] The producer/consumer language should be used as much as possible
- [x] The title slides for the systems (and any other import information) should link to our main website pages. Doing this is a bit tricky since the tutorial is created independently, so links will have to be create as relative html links that just will be broken if the main website is not there, which should be fine since the plan is to just use the website when giving the training.

## Impact

This will add an improved training experience.

### PR DESCRIPTION
**Pull request by GiudGiud**
Closes #13617

see https://github.com/idaholab/moose/pull/24549 for inspiration

### CODE REVIEW

File: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md [hunk 1]
```diff
@@ -193,6 +193,27 @@ during linear iterations.
...
+  type = Steady
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+```
+
+Some examples:
+
+- LU : form the actual Jacobian inverse, useful for small to medium problems but does not scale well
+- Hypre BoomerAMG : algebraic multi-grid, works well for diffusive problems
+- jacobi : preconditions with the diagonal of Jacobian
```
Comment (lindsayad) [inline tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md:211]:
```suggestion
- Jacobi : preconditions with the diagonal, row sum, or row max of Jacobian
```

File: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md [hunk 2]
```diff
@@ -193,6 +193,27 @@ during linear iterations.
...
+
+Can also combine pre-conditioners using sub-preconditioning with option `-sub_pc_type`.
```
Comment (lindsayad) [inline tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md:213]:
```suggestion
For parallel solves, the sub-block (on-process) solvers can be controlled with the PETSc option `-sub_pc_type`. E.g. for a parallel block Jacobi preconditioner (-pc_type jacobi) the sub-block preconditioner could be set to ILU or LU etc. with `-sub_pc_type ilu`, `-sub_pc_type lu`, etc.
```
You can truly combine preconditioners with PETSc using `-pc_type composite`, but we don't really have first class support for this in MOOSE

**Review by lindsayad — COMMENTED** [review]
trying to jam in reviews as I'm being constantly told to refresh

File: tutorials/tutorial02_multiapps/doc/content/getting_started/examples_and_tutorials/tutorial02_multiapps/images/multiapps.pptx
Comment (lindsayad) [inline tutorials/tutorial02_multiapps/doc/content/getting_started/examples_and_tutorials/tutorial02_multiapps/images/multiapps.pptx:1]: checking in a power point?

File: tutorials/tutorial02_multiapps/step03_coupling/02_parent_picard.i
Comment (lindsayad) [inline tutorials/tutorial02_multiapps/step03_coupling/02_parent_picard.i:1]: should this file be renamed?

File: tutorials/tutorial02_multiapps/step03_coupling/03_parent_subcycling_picard.i
Comment (lindsayad) [inline tutorials/tutorial02_multiapps/step03_coupling/03_parent_subcycling_picard.i:1]: same question

File: tutorials/tutorial02_multiapps/doc/content/getting_started/examples_and_tutorials/tutorial02_multiapps/images/multiapps.pptx
Comment (GiudGiud) [reply tutorials/tutorial02_multiapps/doc/content/getting_started/examples_and_tutorials/tutorial02_multiapps/images/multiapps.pptx:1]: updating it. it s already checked in

File: tutorials/tutorial02_multiapps/step03_coupling/02_parent_picard.i
Comment (GiudGiud) [reply tutorials/tutorial02_multiapps/step03_coupling/02_parent_picard.i:1]: no should not
it s ok to talk about picard if you are actually doing picard

File: tutorials/tutorial02_multiapps/doc/content/getting_started/examples_and_tutorials/tutorial02_multiapps/images/multiapps.pptx
Comment (GiudGiud) [reply tutorials/tutorial02_multiapps/doc/content/getting_started/examples_and_tutorials/tutorial02_multiapps/images/multiapps.pptx:1]: I agree this isnt the place...
should we put them in large_media?

Comment (lindsayad) [reply tutorials/tutorial02_multiapps/doc/content/getting_started/examples_and_tutorials/tutorial02_multiapps/images/multiapps.pptx:1]: don't need to do it here in this PR given it's already checked in, but yea that seems like the right place

**Review by loganharbour — APPROVED** [review]

File: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md (hunk 1 above)
Comment (lindsayad) [reply tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md:211]: :-/

File: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md (hunk 2 above)
Comment (lindsayad) [reply tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md:213]: :-/

File: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md (hunk 1 above)
Comment (GiudGiud) [reply tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md:211]: I ll make a follow on

**Pull request merged**